### PR TITLE
Add diagnostic message on project import failure in integration test

### PR DESCRIPTION
**Pull request by chanseokoh**
About #2569. Trying to see what's going on if this fails.

This should log below if all goes right.
```
TreeItem with text {General}: expanded? true
  TreeItem with text {Archive File}
  TreeItem with text {Existing Projects into Workspace}
  TreeItem with text {File System}
  TreeItem with text {Preferences}
```

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/SwtBotAppEngineActions.java [hunk 1]
```diff
@@ -139,8 +139,16 @@ public static IProject importNativeProject(SWTWorkbenchBot bot, String projectNa
 
     SwtBotTreeUtilities.waitUntilTreeHasItems(bot, bot.tree());
     SWTBotTreeItem treeItem = bot.tree().expandNode("General");
-    SwtBotTreeUtilities.waitUntilTreeItemhasChild(bot, treeItem,
-        "Existing Projects into Workspace");
+    try {
+      SwtBotTreeUtilities.waitUntilTreeItemhasChild(bot, treeItem,
+          "Existing Projects into Workspace");
+    } catch (TimeoutException e) {
+      System.out.println(treeItem + ": expanded? " + treeItem.isExpanded());
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/SwtBotAppEngineActions.java]: These seem like useful diagnostics that should be moved into `waitUntilTreeItemHasChild`.  Oh, and the method should have a capitalized `H`.

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/SwtBotAppEngineActions.java]: System.err probably

File: plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotTreeUtilities.java [hunk 2]
```diff
@@ -364,19 +364,27 @@ public boolean test() throws Exception {
...
+          return "Tree item never appeared";
+        }
+  
+        @Override
+        public boolean test() throws Exception {
+          return treeItem.getNodes().contains(childText);
+        }
+      });
+    } catch (TimeoutException e) {
+      System.err.println(treeItem + ": expanded? " + treeItem.isExpanded());
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotTreeUtilities.java]: Would this be better in `getFailureMessage()`?

Comment (chanseokoh) [reply plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotTreeUtilities.java]: Good idea.

**Comment by chanseokoh** [comment]
I made it to start over one more time if the bot can't find the import wizard, hoping that the second time the bot opens the import dialog, the wizard is there. We'll see if this will fix #2569.

**Pull request merged**